src/elements/paginator/common/paginator-common.ts

```diff
@@ -148,7 +149,10 @@ export const SbbPaginatorCommonElementMixin = <T extends AbstractConstructor<Lit
 
     protected renderPrevNextButtons(): TemplateResult {
       return html`
-        <sbb-mini-button-group ?negative=${this.negative} size=${this.size === 's' ? 's' : 'l'}>
+        <sbb-mini-button-group
+          ?negative=${this.negative}
+          size=${isLean() || this.size === 's' ? 's' : 'l'}
```

Comment:
Not sure whether the size should always be synchronized with paginator size? Or what was your thought here?